src/mac.rs

```diff
@@ -514,64 +525,165 @@ impl Security {
     pub fn from_bits(bits: u8) -> Option<Self> {
         match bits {
             0b0 => Some(Security::None),
-            _   => None,
+            _ => None,
         }
     }
 }
 
-
-/// Defines whether PAN ID compression is enabled
-///
-/// Part of [`Header`]. PAN ID compression is currently not supported by this
-/// implementation.
+/// Defines version information for a frame
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
-pub enum PanIdCompress {
-    /// PAN ID compression is disabled
-    Disabled = 0b0,
+pub enum FrameVersion {
+    /// A frame conforming to the 802.15.4-2003 standard
+    Ieee802154_2003 = 0b00,
+    /// A frame conforming to the 802.15.4-2006 standard
+    Ieee802154_2006 = 0b01,
```

Comment:
How sure are you that this is correct?

I don't have access to the standard, unfortunately, and I've been working off the [DW1000 user manual](https://www.decawave.com/wp-content/uploads/2018/09/dw100020user20manual_0.pdf) and that supports IEEE 802.15.4-2011 and claims that `0b01` is the latest version.

I'm not saying what you wrote here is wrong, I'm just confused :-)

Reply:
In the IEEE 802.15.4-2015 standard there is a table where 0b00 is described as "IEEE Std 802.15.4-2003". 0b01 as "IEEE Std 802.15.4-2006", 0b10 as "IEEE Std 802.15.4" and 0b11 as "Reserved". I believe that "IEEE Std 802.15.4" means the current standard.

Reply:
Thanks for the explanation. Let's leave it as it is then.